### DISCUSSION TITLE
It will e enough 16 TB, of bandwidth if t-pot will run 24/24, for 24 months, on the vps?

### DISCUSSION THREAD

**Discussion by xgithx** [telekom-security/tpotce#1683] [Q&A]
I have purchased a vps where the bandwith, is limited to 16TB, for the next 24 months. Can you tell me please, if only for t-pot to run 24/24 for the next 24 months, 16T bandwith should be enough ? Thank you.

**Comment by t3chn0m4g3** [comment]
This depends on the attack volume the IP address is exposed to, the number and types of honeypots you are exposing.
If traffic is a limiting factor you should make sure to remove Ddospot from the `docker-compose.yml`, as it may amount for a lot of traffic.
At the end however It is impossible to even give an estimated guess.
